usecases/MinimumWorkingExample/dodo.py

```diff
@@ -374,3 +385,84 @@ def task_upload_to_openbis():
             'targets': [sample_file_path],
             'clean': [clean_targets],
         }
+
+@create_after(executed='extract_metadata_emodul')
+def task_perform_calibration():
```

Comment:
lets complete this, later this can be adopted in the KG, however they want to store it

Reply:
for them to store it at some point it needs to be computed anyways. I thought it might be better to get this done now, so that we don't need to add other computations later. also, this will be a good thing to "present". showing a long list of samples will confuse many people. however, if you would rather come back to this later, its okay for me.